core/state/statedb.go

```diff
@@ -1171,6 +1171,7 @@ func (s *StateDB) Commit(block uint64, deleteEmptyObjects bool) (common.Hash, er
 		lock    sync.Mutex
 		root    common.Hash
 		workers errgroup.Group
+		set     *trienode.NodeSet
```

Comment:
it might cause some confusion as both `account trie` and `storage tries` will generate a trienode set